src/Invariants/degree_sequence_invariants/annihilation.jl

```diff
@@ -18,9 +18,8 @@ function compute(
     n = Graphs.nv(g)
 
     for i in reverse(1:n)
-        if sum(D[1:i]) <= m
-            return i
-        end
+        isum(D[1:i]) <= m && return i
```

Comment:
I searched `isum` in the REPL's help mode and didn't find anything. Do you mean `sum`?

Reply:
Good catch!